ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/CommandStore.java

```diff
@@ -103,7 +103,7 @@ public QueuedCommandStatus enqueueCommand(final ConfiguredStatement<?> statement
 
     // new commands that generate queries will use the new query id generation method from now on
     final Command command = new Command(
-        statement.getStatementText(),
+        statement.getUnMaskedStatementText(),
```

Comment:
Command statement is going through the parser to run, so we need this unmasked https://github.com/confluentinc/ksql/blob/cb68740012a777c825056bd8e3ac13733db3bb81/ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/StatementExecutor.java#L209

Reply:
Today it doesn't matter because the only two types of (valid) statements which are masked are CREATE CONNECTOR and INSERT VALUES, neither of which is written to the command topic. Once we update the code to have proper handling of masked statements through being written to and read from the command topic, there will be a bunch of other places we'll need changes to. That's planned as follow-up work, right? Only on newer branches, I'm guessing?